src/app.rs

```diff
@@ -127,6 +127,7 @@ pub fn build_app() -> App<'static, 'static> {
                 .short("0")
                 .overrides_with("print0"),
         )
+        .arg(arg("prune").long("prune").short("P"))
```

Comment:
I am very hesitant in introducing new short options (`-P`). Given that `-P` has another meaning in `find`, I would suggest we remove the short option here. Also, please hide this option from the short `-h` text:
```
.hidden_short_help(true)
```